### PR TITLE
The internet herokuapp

### PR DESCRIPTION
**Pull request by lienngo1985**
Upload new document.

### CODE REVIEW

**Comment by lienngo1985** [comment]
Add Document # @

- [ ]

**Comment by globalworming** [comment]
I am a bit lost.. What is the purpose of this Pull Request?

File: serenity-screenplay-webdriver/src/test/java/net/serenitybdd/screenplay/webtests/actions/NavigateToPage.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    }
+
+    public NavigateToPage selectItem(SubCategory item){
+        this.item = item;
+        return this;
+    }
+
+    @Override
+    public <T extends Actor> void performAs(T actor) {
+        waitABit(10000);
```
Comment (wakaleo) [inline serenity-screenplay-webdriver/src/test/java/net/serenitybdd/screenplay/webtests/actions/NavigateToPage.java:40]: A task is performed by an actor, and a UI task typically uses the actor's driver instance. Here you are mixing imperative/Page Object-style code with more functional-style Screenplay tasks, which doesn't really work well. If you need to wait for an element to appear, just get the actor to do the waiting.

File: serenity-screenplay-webdriver/src/test/java/net/serenitybdd/screenplay/webtests/pages/JLHomePage.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    public static Target HRFSITE = Target.the("SiteHref").locatedBy(SITE);
+    public static Target MNUADDSITE = Target.the("AddSiteMenuItem").locatedBy(ADDSITE);
+
+    @FindBy(id = "siteMenu")
+    public WebElementFacade site;
+
+    @FindBy(id = "createSiteMenu")
+    public WebElementFacade addSite;
+
+    public void navigateToPage(String pageName) {
```
Comment (wakaleo) [inline serenity-screenplay-webdriver/src/test/java/net/serenitybdd/screenplay/webtests/pages/JLHomePage.java:27]: If you are doing a screenplay implementation, the navigation logic should be a Task or Interaction, not a Page Object method. In Screenplay, Page Objects are strictly limited to containing locators and locator logic.

**Review by wakaleo — COMMENTED** [review]
Could you give an overview of the purpose of the PR?

File: serenity-screenplay-webdriver/src/test/java/net/serenitybdd/screenplay/webtests/actions/NavigateToPage.java (hunk 1 above)
Comment (lienngo1985) [reply serenity-screenplay-webdriver/src/test/java/net/serenitybdd/screenplay/webtests/actions/NavigateToPage.java:40]: Hi @wakaleo I'm sorry for my mistakes to create this pull request by accidentally. However, I'll appreciate for your kind feedback. Could you please me to close this one? Appreciate and thank you!

File: serenity-screenplay-webdriver/src/test/java/net/serenitybdd/screenplay/webtests/pages/JLHomePage.java (hunk 2 above)
Comment (lienngo1985) [reply serenity-screenplay-webdriver/src/test/java/net/serenitybdd/screenplay/webtests/pages/JLHomePage.java:27]: Hi @wakaleo I'm sorry for my mistakes to create this pull request by accidentally. However, I'll appreciate for your kind feedback. Could you please me to close this one? Appreciate and thank you!

**Pull request closed without merging**